pkg/apis/networking/validation/validation.go

```diff
@@ -351,7 +351,7 @@ func ValidateIngressStatusUpdate(ingress, oldIngress *networking.Ingress) field.
 	return allErrs
 }
 
-// ValidateLIngressoadBalancerStatus validates required fields on an IngressLoadBalancerStatus
```

Comment:
Interesting, this is not using anywhere, right? (unrelated to the PR just noticed that we should remove it then in a follow up or understand why we are not validating this field)

Reply:
It is... it's called as part of validating Ingress objects